bin/post_compile

```diff
@@ -5,6 +5,11 @@ export PATH=/app/.heroku/node/bin:$PATH
 ./manage.py migrate --noinput
 ./manage.py collectstatic --noinput
 
+# Create a file with the current git HEAD revision.
+# We use that file to expose the git revision in our web app, allowing
+# us to use services like whatsdeployed to compare what is deployed to stage and prod for example.
+git rev-parse HEAD > static/revision.txt
```

Comment:
That probably won't work: https://stackoverflow.com/questions/14583282/heroku-display-hash-of-current-commit